fresco.config.ts

```diff
@@ -11,3 +13,11 @@ export const dateOptions: Intl.DateTimeFormatOptions = {
   hour: 'numeric',
   minute: 'numeric',
 };
+
+export const DEFAULT_APP_SETTINGS = {
+  initializedAt: new Date(),
```

Comment:
Should `initializedAt` and `installationId` have default values here? We probably want `installationId` to still be configurable via an env variable (maybe something like: if the env variable exists, that value is used when initialising, and it becomes "read only" in the ui?), and `initializedAt` feels more like it should be calculated as the db insert is happening. What do you think?